src/main/zapHomeFiles/hud/serviceworker.js

```diff
@@ -288,17 +306,28 @@ function showHudSettings(tabId) {
 };
 
 function resetToDefault() {
-	utils.initializeHUD()
-		.then(utils.loadAllTools)
+	utils.loadAllTools()
 		.then(tools => {
 			var promises = [];
+			// Clearing these values means they will go back to the default when next read
+			promises.push(backup('leftPanel', ''));
+			promises.push(backup('rightPanel', ''));
 
 			for (var tool in tools) {
-				promises.push(self.tools[tools[tool].name].initialize());
```

Comment:
Yes, because the initialize() functions dont return a promise, and should therefore not be treated as such.

Reply:
ooh good catch! :D 

we could change the initialize() functions to return utils.writeTool so that they return a promise, allowing us to ensure that these steps get completed, but thats not necessary if there's nothing that follows thats dependent on the tools being initialized. 

I'm a little concerned that there might be something that is dependent on them so by letting them run async we might create a hard to find race condition. 

Thoughts?